### PR TITLE
completed CRUD operations and logic for assign method

### PR DESCRIPTION
**Pull request by Aabid-Faruk**

### CODE REVIEW

**Review by Mohammada72786 — APPROVED** [review]

File: HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java [hunk 1]
```diff
@@ -29,25 +32,58 @@
...
+        return "index";
+    }
+    
+    @RequestMapping("/index")
+    public String indexPage(Model model) {
+    	model.addAttribute("title", "Home");
+        return "index";
+    }
+    
+    @RequestMapping("/recruiter")
```
Comment (Vasanth-I2I) [inline HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java]: Why you have used RequestMapping for every methods? Why you didn't
 used methods like GetMapping, PostMapping and all

File: HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java [hunk 2]
```diff
@@ -74,9 +110,28 @@ public RedirectView createRecruiter(@ModelAttribute RecruiterDTO recruiter, Http
         }
         return redirectView;
     }
+    
+    @RequestMapping(value = "/handle-Applicant", method = RequestMethod.POST)
+    public RedirectView createApplicant(@ModelAttribute ApplicantDTO applicant, HttpServletRequest request) {
```
Comment (Vasanth-I2I) [inline HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java:115]: avoid using "-" in URL

File: HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java [hunk 3]
```diff
@@ -74,9 +110,28 @@ public RedirectView createRecruiter(@ModelAttribute RecruiterDTO recruiter, Http
...
+        System.out.println(applicant);
+        RedirectView redirectView = new RedirectView();
```
Comment (Vasanth-I2I) [inline HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java:116]: Remove Print Statements.. before pushing clean up your code to remove unwanted lines

File: HirePro/src/main/java/com/ideas2it/hirepro/customLogger/CustomLogger.java [hunk 4]
```diff
@@ -13,7 +12,6 @@ public CustomLogger(Class<?> className) {
     }
         
     public static void info(String message) {
-        //logger.info(message);
         logger.info("apple");
```
Comment (Vasanth-I2I) [inline HirePro/src/main/java/com/ideas2it/hirepro/customLogger/CustomLogger.java:15]: What exactly you're doing in the Logger.info method

File: HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java (hunk 1 above)
Comment (Aabid-Faruk) [reply HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java]: I am still reading about all the annotations and the advantages of one over another. That is why i was using RequestMapping as a general tag

File: HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java (hunk 2 above)
Comment (Aabid-Faruk) [reply HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java:115]: ok

File: HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java (hunk 3 above)
Comment (Aabid-Faruk) [reply HirePro/src/main/java/com/ideas2it/hirepro/controller/RecruiterController.java:116]: ok

File: HirePro/src/main/java/com/ideas2it/hirepro/customLogger/CustomLogger.java (hunk 4 above)
Comment (Aabid-Faruk) [reply HirePro/src/main/java/com/ideas2it/hirepro/customLogger/CustomLogger.java:15]: I am having issues with custom logger. its is giving null pointer exception. I ll look into it and update once resolved

**Comment by Vasanth-I2I** [comment]
Add comments for all the methods. pls don't take adding comment as a separate task.

**Pull request closed without merging**